src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClassWithObject.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public class SimpleTestClassWithObject : ITestClass
+    public class SimpleTestClassWithSimpleObject : ITestClass
```

Comment:
Can you add this to the couple areas that use DataMember to enumerate through a bunch of test classes?

Reply:
Will do